onnxruntime/core/platform/windows/telemetry.cc

```diff
@@ -32,7 +32,7 @@ TRACELOGGING_DEFINE_PROVIDER(telemetry_provider_handle, "Microsoft.ML.ONNXRuntim
 
 OrtMutex WindowsTelemetry::mutex_;
 uint32_t WindowsTelemetry::global_register_count_ = 0;
-bool WindowsTelemetry::enabled_ = false;
+bool WindowsTelemetry::enabled_ = true;
```

Comment:
This will be reset back to false.

Reply:
Still needs to be reset back to false.

---
In reply to: [355012904](https://github.com/microsoft/onnxruntime/pull/2577#discussion_r355012904) [](ancestors = 355012904)